### DISCUSSION TITLE
Pemision denied(publickey error)

### DISCUSSION THREAD

**Discussion by Mo-Mosala** [community/community#33262] [Other Feature Feedback, Questions, & Ideas]
Hello, I keep on getting permission denied error while trying to login from Git bash. The public key that I run on the agent is the same as the one on the portal. I tried running ssh over https but I still cant login. Is there something that I am missing? Thank you for you help.

**Comment by airtower-luna** [comment]
You'll need to share the *exact and full* commands and error messages to get a useful response (you can censor private repository names, etc.).

In general, if you're trying to connect with SSH you can use `ssh -v git@github.com` to get debug info on the SSH connection itself.
